temporalio/bridge/worker.py

```diff
@@ -102,3 +104,35 @@ def retry_policy_from_proto(
         maximum_attempts=p.maximum_attempts,
         non_retryable_error_types=p.non_retryable_error_types,
     )
+
+
+def from_bridge_payloads(
+    payloads: Iterable[temporalio.bridge.proto.common.Payload],
```

Comment:
@Sushisource we should have just used the API definition of Payload, lang shouldn't need to bother with these conversions.